app/services/breadcrumbs_options.rb

```diff
@@ -30,11 +30,15 @@ def index_item
     end
 
     {
-      caption: _(resource_name.pluralize.humanize),
-      url: resource_path(class_name)
+      caption: get_caption(resource_name),
+      url: resource_path(class_name) || resource_path(resource_name)
     }
   end
 
+  def get_caption(resource_name)
+    (resource_name == 'operatingsystem') ? _('Operating Systems') : _(resource_name.pluralize.titleize)
```

Comment:
probably a more generic solution is needed, it affect more resources:
![hostgroup](https://user-images.githubusercontent.com/11807069/43077772-2625f866-8e91-11e8-8950-576e193b766c.png)

Reply:
I agree, do you have something in mind?
for now I updated with case statement